fdbclient/ServerKnobs.cpp

```diff
@@ -644,6 +644,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( MAX_STORAGE_COMMIT_TIME,                             120.0 ); //The max fsync stall time on the storage server and tlog before marking a disk as failed
 	init( RANGESTREAM_LIMIT_BYTES,                               2e6 ); if( randomize && BUGGIFY ) RANGESTREAM_LIMIT_BYTES = 1;
 	init( ENABLE_CLEAR_RANGE_EAGER_READS,                       true );
+	init( ENABLE_PTREE_LOOKUP_DURING_EAGER_READS,				true );
```

Comment:
I'll defer to Steve, but if this passes simulation, I don't think we need this knob. If we do need it, nit: formatting.